src/flycapture2.pyx

```diff
@@ -385,6 +394,165 @@ cdef class Context:
         with nogil:
             r = fc2SetConfiguration(self.ctx, &config)
         raise_error(r)
+        
+        
+    ############################################################
+    # GigE support
+
+    def set_byteorder_littleendian(self):
+        '''Get byte order        
+        '''
+        cdef fc2Error r
+        cdef unsigned int regval = 0        
+        
+        with nogil:    
+            r = fc2WriteRegister(self.ctx, 0x1048, regval)
+        raise_error(r)
+        
+    def set_byteorder_bigendian(self):
+        '''Get byte order        
+        '''
+        cdef fc2Error r
+        cdef unsigned int regval = 1<<31      
+        
+        with nogil:    
+            r = fc2WriteRegister(self.ctx, 0x1048, regval)
+        raise_error(r)
+
+        
+    def rescan_bus(self):
+        '''Rescans the bus to discover new devices.
+        '''
+        cdef fc2Error r
+        with nogil:
+            r = fc2RescanBus(self.ctx)
+        raise_error(r)
+        
+    def get_gige_cams(self):
+        '''Returns a list of the serial numbers of the connected GigE cameras.
+        '''
+        cdef fc2Error error
+        cdef fc2Error r
+        cdef fc2CameraInfo cams[8]
+        cdef fc2CameraInfo *pcams = NULL
+        cdef unsigned int count = sizeof(cams)
+        cdef int i
+
+        with nogil:
+            error = fc2DiscoverGigECameras(self.ctx, cams, &count)
+        if error == FC2_ERROR_BUFFER_TOO_SMALL:
+            pcams = <fc2CameraInfo *>malloc(count * sizeof(fc2CameraInfo))
+            if pcams == NULL:
+                raise MemoryError()
+
+            try:
+                with nogil:
+                    r = fc2DiscoverGigECameras(self.ctx, pcams, &count)
+                raise_error(r)
+                return [pcams[i].serialNumber for i in range(count)]
```

Comment:
This is the discover_gige_cameras() which return the serial number of the cameras found. No need to return something else.